internal/process/cluster.go

```diff
@@ -19,25 +19,25 @@ func createClusterSamples(config *config.Config) {
 
 			//Retrieving the list of host belonging to the cluster
 			hostList := ""
-			for _, hostReference := range cluster.Host {
-				if host, ok := dc.Hosts[hostReference.Reference()]; ok {
-					hostList = hostList + host.Summary.Config.Name + "|"
+			for _, hr := range cluster.Host {
```

Comment:
To old sintax wasn't the standard one to check if a map had a key?

Reply:
Yes, I just don't like to expose those "internal" objects to the outside. But to honest we do use them in other places so this doesn't help that much.
Nevertheless in the future I think we should try to avoid exposing "objects" like this. It makes it very easy to change the values and add bugs to the code.